src/CMake/FindVisItPython.cmake

```diff
@@ -101,7 +104,13 @@ find_package(PythonInterp REQUIRED)
 if(PYTHONINTERP_FOUND)
         
         MESSAGE(STATUS "PYTHON_EXECUTABLE ${PYTHON_EXECUTABLE}")
-        
+
+        execute_process(COMMAND "${PYTHON_EXECUTABLE}" "-c" 
+                        "import sys;from distutils.sysconfig import get_config_var; sys.stdout.write(get_config_var('VERSION'))"
+                        OUTPUT_VARIABLE PYTHON_CONFIG_VERSION
+                        ERROR_VARIABLE  ERROR_FINDING_PYTHON_VERSION)
+        MESSAGE(STATUS "PYTHON_CONFIG_VERSION $PYTHON_CONFIG_VERSION}")
```

Comment:
Need an opening brace in the status message for PYTHON_CONFIG_VERSION.

Reply:
- [ ] cyrus fix this